### ISSUE TITLE
Posgres connection test should verify that the Row sampling extension is loaded.

### ISSUE THREAD

**Issue by lloydtabb** [looker-open-source/malloy#577]
We have to create the extension (which works in GoogleSQL by default).

https://www.postgresql.org/docs/current/tsm-system-rows.html

**Comment by mtoy-googly-moogly** [comment]
Yeah I test against a local postgres instance because then I am not in credential hell. A step by step "how to install the needed extension on your postgres" would be great for me, and for people trying to use Malloy.
